packages/factory/README.md

```diff
@@ -82,6 +83,13 @@ Most of the time they will interact via the global context.
 
 #### [📝 Full documentation ➡️](/packages/plugins/injection#readme)
 
+
+### True End
```

Comment:
I find the name a bit weird (but I don't have a better suggestion :/)

what about `onFullBuild`?

Reply:
I find `onFullBuild` a bit misleading too, for instance in `rollup` the concept of `buildEnd` happens before the `writeBundle`, as-in, rollup (and vite) separate both the build from the write process.

Can you elaborate on what you find weird with "true end"?
Maybe we could go with just `end`?

Reply:
"true" doesn't really have a definition. And it sounds a bit more like something you'd say when speaking to someone vs something defined in a spec / technical document

Reply:
After some discussion offline, it's difficult to find a term that would really define this hook, so for now, until we get better consensus, we'll stick with `true-end`.